index.d.ts

```diff
@@ -21,27 +21,208 @@ declare module 'react-native-health' {
   export interface AppleHealthKit {
     initHealthKit(
       permissions: HealthKitPermissions,
-      callback: (error: string, result: Object) => void,
-    ): void
-
-    saveFood(
-      options: Object,
-      callback: (error: string, result: Object) => void,
+      callback: (error: string, result: HealthValue) => void,
     ): void
 
     isAvailable(callback: (error: Object, results: boolean) => void): void
 
+    // initStepCountObserver
```

Comment:
Looks like we could probably remove this comment, right?

Reply:
Yeah, that can be removed :)